robottelo/cli/hammer.py

```diff
@@ -49,8 +49,9 @@ def parse_csv(output):
     # ignore warning about puppet and ostree deprecation
     output.replace('Puppet and OSTree will no longer be supported in Katello 3.16\n', '')
     is_rex = True if 'Job invocation' in output else False
+    is_pkg_list = True if 'Nvra' in output else False
     # Validate if the output is eligible for CSV conversions else return as it is
-    if not is_csv(output) and not is_rex:
+    if not is_csv(output) and not (is_rex or is_pkg_list):
```

Comment:
I would keep the expression simple...
`if not is_csv(output) and not is_rex and not is_pkg_list:`
or
`if not (is_csv(output) or is_rex or is_pkg_list):`